q2_longitudinal/tests/test_longitudinal.py

```diff
@@ -303,7 +331,80 @@ def test_volatility(self):
         volatility(
             output_dir=self.temp_dir.name, metadata=self.md_ecam_fp,
             metric='observed_otus', group_column='delivery',
-            state_column='month', individual_id_column='studyid')
+            state_column='month', individual_id_column='studyid',
+            spaghetti='yes')
+
+    def test_volatility_no_spaghetti_no_control(self):
+        volatility(
+            output_dir=self.temp_dir.name, metadata=self.md_ecam_fp,
+            metric='observed_otus', group_column='delivery',
+            state_column='month', individual_id_column='studyid',
+            spaghetti='no', plot_control_limits=False)
```

Comment:
In a perfect world we'd have something about the visualization we could test, but I can't come up with anything, so this is definitely better than nothing.

Reply:
Yeah... there might be some way to extract the line coordinates or something but it would probably get really ugly.